guide/install.md

```diff
@@ -128,6 +128,13 @@ system will boot successfully even when the hard drive is unavailable (such as
 when your external USB storage is unmounted). This can be helpful when
 recovering from problems.
 
+Create the sample directory.
+
+```
+sudo mkdir /media/nvr/sample
+sudo chmod a+rw -R /media/nvr
```

Comment:
`sudo chown moonfire-nvr:moonfire-nvr /media/nvr` instead? Nothing other than Moonfire NVR itself should be messing with these files.

Reply:
Yeah, that probably makes more sense